adb/contrib/high.py

```diff
@@ -729,7 +729,7 @@ def GetLastUID(self):
 
   def GetPackages(self):
     """Returns the list of packages installed."""
-    out, _ = self.Shell('pm list packages')
+    out, _ = self.Shell('pm list packages', timeout_ms=30000)
```

Comment:
HighDevice does not store a default timeout.  Added the max(timeout, default) logic to AdbCommandsSafe and to common.Handle.